oozie-to-airflow/utils/el_utils.py

```diff
@@ -156,8 +156,20 @@ def parse_els(properties_file: str, prop_dict: Dict[str, str] = None):
                     if line.startswith("#") or line.startswith(" ") or line.startswith("\n"):
                         continue
                     else:
-                        props = [x.strip() for x in line.split("=", 1)]
-                        prop_dict[props[0]] = replace_el_with_var(props[1], prop_dict, quote=False)
+                        _convert_line(line, prop_dict)
         else:
             logging.warning(f"The job.properties file is missing: {properties_file}")
     return prop_dict
+
+
+def _convert_line(line: str, prop_dict: Dict[str, str]) -> None:
+    """
+    Converts a line from the properties file and adds it to the properties dictionary.
+    If the line is a comma-separated list of items, it will be added as a Python list.
+    If it's a single item, it will be added as a string.
+    """
+    props = [x.strip() for x in line.split("=", 1)]
```

Comment:
I do not want you to make changes, but this loop makes it difficult to understand this code. 
My proposition
```python
    key, value = line.split("=", 1)
    value = replace_el_with_var(value.strip(), prop_dict, quote=False)
    values = value.split(",")
    prop_dict[key.strip()] = values[0] if len(values) <= 1 else values
```

Reply:
This loop was there before, but good idea - your proposition is clearer. I'll change it.